UI/src/assetTypeConfiguration.js

```diff
@@ -478,7 +478,7 @@
           {'name': "Tarkistettu", 'propType': 'checkbox', 'id': "huoltotie_tarkistettu", value: [{typeId: 0, title: 'Ei tarkistettu'}, {typeId: 1, title: 'Tarkistettu'}]}],
         style: new ServiceRoadStyle(),
         label : new ServiceRoadLabel(),
-        isVerifiable: true
+        isVerifiable: false
```

Comment:
It's suppose to change this, or was a mistake when testing?

Reply:
no, even though the asset is verifiable it was not included in the user story for the verification button. so the button did nothing before.